### DISCUSSION TITLE
【求助】蛤蟆泛滥怎么办

### DISCUSSION THREAD

**Discussion by keerdun** [share121/inter-knot#1257] [其他]
![IMG_20240810_165425](https://github.com/user-attachments/assets/c1042420-e550-4679-837b-fad374eb1fdc)
 最近发现家里突然出现了好多黑色的小蛤蟆，大概和大拇指肚一般大，但往年都没见过这些玩意，求个前因后果，或说各位有没有相同情况出现
 报酬：20菲林

**Comment by GWWA** [comment]
这个委托我接了

**Comment by Panjk8** [comment]
这不好解决吧
